config/initializers/figaro.rb

```diff
@@ -1,15 +1,3 @@
 # Put here the variables used by all the environments
 variables = %w[SERVER_URL PASSWORD_RESET_URL]
-
-unless Rails.env.test?
-  # Variables not used by the test environment
-  variables += %w[SECRET_KEY_BASE
```

Comment:
We should keep `SECRET_KEY_BASE` because it's not related to any external service.

Maybe the best approach is to keep the structure of the code and just delete the variables below.